tests/aggregation/tests.py

```diff
@@ -754,18 +822,18 @@ def test_order_of_precedence(self):
 
     def test_combine_different_types(self):
         with six.assertRaisesRegex(self, FieldError, 'Expression contains mixed types. You must set output_field'):
-            Book.objects.annotate(sums=Sum('rating') + Sum('pages') + Sum('price')).get(pk=4)
+            Book.objects.annotate(sums=Sum('rating') + Sum('pages') + Sum('price')).get(isbn="013235613")
```

Comment:
What about using `(pk=cls.b4.pk)` instead? I think it would be easier to refer to the source of the data this way.